### PR TITLE
Modifies test relying on mesos logging in the stdout of a task

### PR DESCRIPTION
**Pull request by sambatyon**
## High-level description

Mesos was modified so that it no longer logs information on tasks standard output, however, some tests relied in this functionality to verify the api to check the output of tasks.

The quick fix was to modify the task definition so that it writes something into its own standard output.

Note that this PR does not change the behavior of the current master, however no mesos bump can be done until this test is modified since all its bumps will fail on  `test_mesos.test_files_api()`.

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS_OSS-3558](https://jira.mesosphere.com/browse/DCOS_OSS-3558) Nightly Mesos bump breaks DC/OS tests due to expected stdout logging..

## Checklist for all PRs

  - [x] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change: It modifies a unit test.
  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here: The change is in the test.
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by mesosphere-mergebot-dev** [comment]
This repo has @mesosphere-mergebot integration. You can interact with the following commands.
```
@mesosphere-mergebot bump-ee  
@mesosphere-mergebot merge-it  
@mesosphere-mergebot override-status  
@mesosphere-mergebot label [Work In Progress |Ready For Review |Holding |Ship It |Request For Comment] 
```

* PR creators can apply one of  `[Ready For Review |Work In Progress]`. [Owners](https://github.com/dcos/dcos/blob/master/owners.json) can apply any label.

**Comment by sambatyon** [comment]
@mesosphere-mergebot bump-ee

**Comment by sambatyon** [comment]
@mesosphere-mergebot label Ready For Review

File: packages/dcos-integration-test/extra/test_mesos.py [hunk 1]
```diff
@@ -137,6 +137,7 @@ def _post_agent(url, headers, json=None, data=None, stream=False):
 
 def test_files_api(dcos_api_session):
     app, test_uuid = test_helpers.marathon_test_app()
+    app['cmd'] = 'echo $DCOS_TEST_UUID && ' + app['cmd']
```
Comment (rukletsov) [inline packages/dcos-integration-test/extra/test_mesos.py]: It looks like github does not allow me to comment on the lines below, so I'm leaving the comment here. For an outside observer it is not directly clear what this test is doing. As discussed online, please add a comment either at the beginning of the test or below, explaining that:
* the test basically checks that both stdout and stderr are not empty;
* Mesos is expected to write to stderr only.

File: packages/dcos-integration-test/extra/test_mesos.py [hunk 2]
```diff
@@ -136,7 +136,14 @@ def _post_agent(url, headers, json=None, data=None, stream=False):
 
 
 def test_files_api(dcos_api_session):
+    '''
+    This test verifies that the standard output and error of a Mesos task can be
+    read. We check that neither standard output or error are empty files. Since
```
Comment (tillt) [inline packages/dcos-integration-test/extra/test_mesos.py]: s/or/nor/

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
@sambatyon  - The `bump-ee` wasn't done after the last update on https://github.com/dcos/dcos/pull/2940/commits/c19174d24564558c9392037fdc29c73ce104cf49 

1) The master had moved as fix for tox test.
2) I updated this branch and did the `bump-ee`.

**Comment by orsenthil** [comment]
@mesosphere-mergebot override-status teamcity/dcos/test/azure/arm https://jira.mesosphere.com/browse/DCOS_OSS-3539

**Comment by orsenthil** [comment]
@mesosphere-mergebot override-status teamcity/dcos/test/dcos-docker/static https://jira.mesosphere.com/browse/DCOS_OSS-2115

**Comment by orsenthil** [comment]
Thanks @amitaekbote !

**Pull request merged**